.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+<!-- Please create/claim an issue before sending a PR -->
+<!-- Add issue number (Eg: fixes #123) -->
+
+Fixes #
+
+### Checklist
+- [ ] My branch is up-to-date with upstream/master branch.
+
+### Description
+<!-- Describe about what this PR does, previous state and new state of the output -->
+
+### Change logs
+
+<!-- #### Added -->
+<!-- Edit these points below to describe the new features added with this PR -->
+<!-- - Feature 1 -->
+<!-- - Feature 2 -->
+
+
+#### Changed

```

Comment:
this is not needed, you have written change logs above

Reply:
which one should I remove changed or change logs?

Reply:
resolved.